python/tvm/relay/op/strategy/hexagon.py

```diff
@@ -21,7 +21,7 @@
 from tvm import topi
 from .generic import *
 from .. import op as _op
-
+from .generic import is_depthwise_conv2d
```

Comment:
There is already an import of `*` from `.generic` at line 22.

Reply:
done.